api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -336,6 +337,14 @@ def serialize_offer(cls, offer: offers_models.Offer) -> dict:
         distinct = extra_data.get("isbn") or extra_data.get("visa") or str(offer.id)
         distinct += extra_data.get("diffusionVersion", "")
 
+        music_type_label = None
+        music_type = (offer.extraData or {}).get("musicType", "").strip()
```

Comment:
(détail) `extra_data` est déjà défini plus haut : 

```suggestion
        music_type = extra_data.get("musicType", "").strip()
```

Reply:
bien vu